modules/caddyhttp/caddyauth/caddyauth.go

```diff
@@ -78,6 +78,9 @@ func (a Authentication) ServeHTTP(w http.ResponseWriter, r *http.Request, next c
 
 	repl := r.Context().Value(caddy.ReplacerCtxKey).(*caddy.Replacer)
 	repl.Set("http.authentication.user.id", user.ID)
+	for k, v := range user.Metadata {
+		repl.Set("http.authentication.user.metadata." + k, v)
```

Comment:
> modules/caddyhttp/caddyauth/caddyauth.go:82: File is not `goimports`-ed (goimports)

Go home reviewdog, you're drunk

Reply:
I was wondering what it's complaining about. This is the `goimports` diff 😄 

```diff
--- caddyauth.go.orig   2020-03-23 01:23:26.000000000 +0300
+++ caddyauth.go        2020-03-23 01:23:26.000000000 +0300
@@ -79,7 +79,7 @@
        repl := r.Context().Value(caddy.ReplacerCtxKey).(*caddy.Replacer)
        repl.Set("http.authentication.user.id", user.ID)
        for k, v := range user.Metadata {
-               repl.Set("http.authentication.user.metadata." + k, v)
+               repl.Set("http.authentication.user.metadata."+k, v)
        }
 
        return next.ServeHTTP(w, r)
@@ -95,7 +95,7 @@
 
 // User represents an authenticated user.
 type User struct {
-       ID string
+       ID       string
        Metadata map[string]string
 }
```

Reply:
So... goimports is doing fmt now?

Reply:
🤷‍♂ I thought it only sorted imports

Reply:
Haha me too.

Okay, well this PR looks good to me, except for the formatting omissions.

Reply:
https://godoc.org/golang.org/x/tools/cmd/goimports

> In addition to fixing imports, goimports also formats your code in the same style as gofmt so it can be used as a replacement for your editor's gofmt-on-save hook. 

Apparently yes